corehq/apps/analytics/views.py

```diff
@@ -0,0 +1,13 @@
+from django.views.decorators.http import require_POST
+from corehq.apps.analytics.tasks import track_clicked_deploy_on_hubspot
+from django.http import HttpResponse
+
+
+@require_POST
+def hubspot_click_deploy(request):
```

Comment:
http://ccbv.co.uk/ is a good resource for class based views, if you want to know more :)